src/utils/trust.details.ts

```diff
@@ -204,7 +204,7 @@ const getUrl = (isUpdate: boolean) => {
 
 const getNextPage = (isUpdate: boolean, trustId: string) => {
   if (isUpdate){
-    return config.UPDATE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_URL;
+    return config.UPDATE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_URL + "/" + trustId;
```

Comment:
return `${config.UPDATE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_URL} /${trustId}`

Reply:
registration add "/trust-involved" after trust id. Should we be following that pattern?

Reply:
Would probably be best if we keep it as consistent as possible? I'll make the change

Reply:
Fixed